test/test_files/copy/copy_gcs.test

```diff
@@ -0,0 +1,30 @@
+-GROUP CopyGcsTest
+-DATASET CSV tinysnb
```

Comment:
Change "tinysnb" to "empty", since we want to start from an empty db from scratch here.